modules/transport-netty4/src/main/java/org/elasticsearch/http/netty4/Netty4HttpServerTransport.java

```diff
@@ -149,6 +159,33 @@ public Netty4HttpServerTransport(
         ClusterSettings clusterSettings,
         SharedGroupFactory sharedGroupFactory,
         Tracer tracer
+    ) {
+        this(
+            settings,
+            networkService,
+            threadPool,
+            xContentRegistry,
+            dispatcher,
+            clusterSettings,
+            sharedGroupFactory,
+            tracer,
+            TLSConfig.noTLS(),
+            null
```

Comment:
nit: I'd prefer to remove the original constructor and update the usages. The number of tests affected is not too bad and it is just simpler in my mind to reason about being explicit in the callers about noTLS. Completely optional ofc and also fine in a follow up.